core/src/View/HUD.java

```diff
@@ -0,0 +1,132 @@
+package View;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.graphics.g2d.NinePatch;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Stack;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.utils.viewport.FitViewport;
+
+public class HUD {
+    private Stage stage;
+    private FitViewport stageViewport;
+
+    public HUD(SpriteBatch spriteBatch) {
+        stageViewport = new FitViewport(800,480);
+        stage = new Stage(stageViewport, spriteBatch);
+
+        int score = 10;
+
+        //The code under is a way to make a certain color since libgdx somehow doesnt know rgb values needed
+        //for a square (and also make it an actor so I can send it back and not draw directly).
+
+        Texture black= new Texture(Gdx.files.internal("HudColors/Black.png"));
+        NinePatch blackHp = new NinePatch(black, 0, 0, 0, 0);
+        Image blackHpBar = new Image(blackHp);
+        Image blackXpBar = new Image(blackHp);
+
+        Texture aqua = new Texture(Gdx.files.internal("HudColors/Aqua00ffff.png"));
+        NinePatch emptyxp = new NinePatch(aqua, 0, 0, 0, 0);
+        Image xpbar = new Image(emptyxp);
+        /*
+        Texture aquamarine = new Texture(Gdx.files.internal("HudColors/Aquamarine7fffd4.png"));
+        NinePatch fullexperience = new NinePatch(aquamarine, 0, 0, 0, 0);
+        Image fullxpbar = new Image(fullexperience);
+        */
+        Texture green = new Texture(Gdx.files.internal("HudColors/Green00933b.png"));
+        NinePatch fullHp= new NinePatch(green, 0, 0, 0, 0);
+        Image fullhp = new Image(fullHp);
+
+        Texture yellow = new Texture(Gdx.files.internal("HudColors/Yellowf1c50c.png"));
+        NinePatch midHp = new NinePatch(yellow, 0, 0, 0, 0);
+        Image midhp = new Image(midHp);
+
+        Texture red = new Texture(Gdx.files.internal("HudColors/Red00923f.png"));
+        NinePatch lowHp = new NinePatch(red, 0, 0, 0, 0);
+        Image lowhp = new Image(lowHp);
+
+        Texture xpyellow = new Texture(Gdx.files.internal("HudColors/Xpyellowc8a72b.png"));
+        NinePatch xpyellow2 = new NinePatch(xpyellow, 0, 0, 0, 0);
+        Image fullxpbar = new Image(xpyellow2);
+
+        Label hpLabel = new Label("HP", new Label.LabelStyle((new BitmapFont()), Color.WHITE));
+        Label scoreLabel = new Label("Score: " + Integer.toString(score), new Label.LabelStyle((new BitmapFont()), Color.WHITE));
+
+        //3 tables. Id like to have one but stack is not working like I think it should. Currently
+        // it acts as 3 layers, black under, secondary color, main color, text? as a fourth if I want
```

Comment:
If the tables act as three different layers then maybe they could have better names than table1, table2 and table3.